linear-base.cabal

```diff
@@ -37,8 +37,8 @@ library
     Data.Either.Linear
     Data.Eq.Linear
     Data.Functor.Linear
-    Data.Functor.Linear.Internal
-    Data.Functor.Linear.Internal.Traversable
```

Comment:
I don't think we should lose the word `Linear`. Although I am not sure if it should be `Data.Functor.Internal.Linear` or `Data.Functor.Linear.Internal`.

Reply:
This may seem a bit long, but I''ll change it to these:
- `Data.Functor.Linear.Internal.Functor`
- `Data.Functor.Linear.Internal.Applicative`

... since the most specific thing about these internal modules is the _concept_ they contain and the concept of `Applicative` is qualified, say, by being data, being a functor, being linear, and the module being internal.

In terms of interfaces, things like`Data.Functor.Linear` make sense as linear flavors of familiar concepts. But I think internal modules shouldn't follow the same pattern and instead have the last name be one central concept so that it's clear when we import them internally what chiefly we're importing.